packages/core/forms/src/index.ts

```diff
@@ -10,7 +11,7 @@ export default {
 }
 
 export { customFields } from './components/fields/exports'
-export { VueFormGenerator, sharedForms }
+export { VueFormGenerator, sharedForms, FieldTester }
```

Comment:
question: why are we exporting the `FieldTester` component? Should it only be used in the tests?

Reply:
I want to use it the sandbox (for debug) and the tests, but couldn't figure out another way without ending up with 2 copies.

Reply:
Just import it in the tests by relative path; you don't want to export it or even import it at the package root.